src/components/Menu/styles.js

```diff
@@ -1,62 +1,110 @@
+import { css } from 'emotion'
+import facepaint from 'facepaint'
 
+const breakpoints = [576, 768, 1024, 1200]
+const mq = facepaint(breakpoints.map(bp => `@media (min-width: ${bp}px)`))
 
-export const styles = {
-  logoWrap: {
+//export const styles = {
+  export const logoWrap = mq({
     marginLeft: '0!important',
-    paddingTop: '24px!important',
+    paddingTop: '13px!important',
     paddingBottom: '0!important',
     paddingRight: '0!important',
-    paddingLeft: '0!important',
-  },
-  logoImage: {
-    width: 168.64,
-    height: 21.29,
-    marginBottom: 0,
+    paddingLeft: ['15px!important','15px!important','15px!important','10px!important'],
+    fontSize: ['1em','1.5em','2.1em','2.1em'],
+  })
+
+  export const logoLink = css ({
+    textDecoration: 'none!important',
+    color: 'black!important',
+    })
+
+    export const subMenuCol = mq({
+    '& > :nth-child(2)': {
+      paddingTop: ['40px','40','0','0'],
```

Comment:
@Alphascythian can you please, before submitting, make sure the layout it's actually correct and have the same metrics as the provided design specs? If you inspect the element, you'll see that the distance of `40px` it's to much because the element on the top already have a `15px` padding bottom, so the total distance will be `55px`.
The key take away of this is to always double check the output of your code.